src/domain/models/path-templates.ts

```
@@ -116,14 +118,25 @@ export function detectProjectRoot(startPath?: string): string {
-				}
-			} catch {
-				// Continue searching if file check fails
-				//TODO: how about we have some tracking debug/warning here???
+			const fileCheckResult = Effect.runSync(
+				Effect.gen(function* () {
+					// Use basic file system check with Effect.TS Railway programming
+					if (existsSync(configPath)) {
+						return true
+					}
+					return false
+				}).pipe(
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)